doc/manual/manual.tex

```diff
@@ -7795,6 +7795,43 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 
 We can observe that when the two-layer viscosity model is added, there is only one apparent peak for each case. The first peaks of the 0.01 viscosity contrast tests appear earlier and are larger in magnitude than those of 0.1 viscosity contrast tests.  There are no secondary plumes and the whole system tends to reach stability after around 500 time steps.
 
+
+\subsubsection{The Bunge et al.~mantle convection experiments}
+\label{sec:bunge_et_al_cookbook}
+
+\textit{This section was contributed by Cedric Thieulot and Bob Myhill.}
+
+Early mantle modelling studies of the 1970s, 1980s and 1990s were often concerned with simple set-ups (Cartesian geometries, incompressible fluid, free slip boundary conditions) and investigated the influence of the Rayleigh number, the heating mode or the temperature dependence of the viscosity on temperature, pressure and/or strain rate \cite{youn74,buss75,buss79,BBC89,BC93,vavy93,burb97}. In this cookbook, we use the `simple' material model to reproduce the set-up in \cite{burb96}, which reported that even modest increases in mantle viscosity with depth could have a marked effect on the style of mantle convection.
+
+Although the original article showcases results obtained in a 3D hollow sphere, we here run the models in an annular domain of inner radius $R_i = 3480$ \si{\kilo\meter} and outer radius  $R_o = 6370$ \si{\kilo\meter}.
```

Comment:
Here and in many places below: I am a bit confused, in the rest of the manual we have the ``\si{...}`` inside the math environments. Is there a reason you are not doing this? Does this look the same?

Reply:
The units themselves look identical inside or outside the math environment (because \si overrides the local environment).  The spacing between number and \si{unit} is not necessarily the same, since the math environment typically imposes narrower whitespace. I prefer having \si outside the math environment, which consistently imposes a normal-width space between number and unit. 

The functions provided by siunitx always work fine outside the math environment, and mostly work inside the math environment. There are a few functions that only work outside the math environment. For example, the \numlist "function should be used in text mode, a common feature of all of the list and range functions provided by siunitx.". 

As an aside, there are some pretty odd uses of \si in the ASPECT manual right now ... like this one:
$\frac{\textrm{J}}{\si{kg}\cdot\si{K}} = \frac{\si{m}^2}{\textrm{s}^2\cdot\si{K}}$.

which I would write as \si{\joule\per\kilo\gram\per\kelvin} = \si{\meter\squared\per\second\squared\per\kelvin}.

Reply:
An even better option (for another PR) would be to use the \SI{<value>}{<unit>} function, which would negate the need for many $$.

Reply:
> there are some pretty odd uses of \si in the ASPECT manual right now

Yes, for sure.

Reply:
Feel free to fix whatever you find!

Reply:
These particular cases are now fixed in #3832.